### ISSUE TITLE
[Bug]: MSBUILD : error MSB1003: Specify a project or solution file.

### ISSUE THREAD

**Issue by fataIlty** [winstxnhdw/lc-hax#411]
### What happened?

Hello, i downloaded all requirements and then i opened requirements.bat and this error has appeared :
MSBUILD : error MSB1003: Specify a project or solution file.

### Current Commit Hash

I cant find HEAD

### Injector

- [X] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have confirmed that my anti-virus is not blocking any of the relevant programs
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment]
https://github.com/winstxnhdw/lc-hax/discussions/389

**Comment by winstxnhdw** [comment]
Is your `submodules/SharpMonoInjectorCore` folder is empty? If it's not empty, check your anti-virus. But it is more likely that it is empty. This means your recursive clone failed. It can only fail if your Git install is broken. Your issue might be similar to the guy in #293.

**Comment by fataIlty** [comment]
Yeah its empty, i was tryed to open launch.bat and i got error : Can't find "./submodules/SharpMonoInjectorCore/dist/SharpMonoInjector.exe"
Yes,i cloned repository and read note

**Comment by winstxnhdw** [comment]
> I cant find HEAD

If you can't find the HEAD, it means you never cloned the repository.
